contracts/src/NativeTokenDestination.sol

```diff
@@ -122,31 +122,25 @@ contract NativeTokenDestination is
      * @param settings Constructor settings for this destination token bridge instance.
      * @param nativeAssetSymbol The symbol of the native asset.
      * @param initialReserveImbalance The initial reserve imbalance that must be collateralized before minting.
-     * @param decimalsShift The number of decimal places to shift the token amount by.
-     * @param multiplyOnDestination See {TeleporterTokenDestination-multiplyOnDestination}.
      * @param burnedFeesReportingRewardPercentage_ The percentage of burned transaction fees
      * that will be rewarded to sender of the report.
      */
     constructor(
         TeleporterTokenDestinationSettings memory settings,
         string memory nativeAssetSymbol,
         uint256 initialReserveImbalance,
-        uint8 decimalsShift,
-        bool multiplyOnDestination,
         uint256 burnedFeesReportingRewardPercentage_
     )
         ERC20(string.concat("Wrapped ", nativeAssetSymbol), nativeAssetSymbol)
-        TeleporterTokenDestination(
-            settings,
-            initialReserveImbalance,
-            decimalsShift,
-            multiplyOnDestination
-        )
+        TeleporterTokenDestination(settings, initialReserveImbalance)
     {
         require(
             initialReserveImbalance != 0, "NativeTokenDestination: zero initial reserve imbalance"
         );
-
+        require(
+            settings.tokenDecimals == 18, // to match the ERC20 default
```

Comment:
This is to match the EVM's native asset denomination, not related to ERC20s 😄

Reply:
I wrote that because I saw that this constructor is invoking the base contract constructor `ERC20`, and that `ERC20` contract says `The default value of {decimals} is 18. To select a different value for {decimals} you should overload it.`

Reply:
Gotcha, just pointing out that we want to have this requirement of `settings.tokenDecimals == 18` because the native token of an EVM always has a denomination of 18. For `ERC20Destination` you can define your own decimals value and override the default, but for `NativeTokenDestination` you should not be able to.

Reply:
I fixed up this comment in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/d5bb1f79654210a0687da67b2d0abfefc088c1d2